doc/plutus/howtos/exporting-a-script.rst

```diff
@@ -13,6 +13,8 @@ For example, you might want to submit it as part of a manually created transacti
 Fortunately, it is quite simple to do this.
 You can either translate directly into CBOR using the `Serialise` typeclass from the `serialise` package, or you can create an envelope of the sort used by the Cardano node CLI.
 
+The same can also be done with `Datum` and `Redeemer` datatypes.
```

Comment:
Maybe change the title of the doc since it includes more stuff now?

Reply:
exporting-data? exporting-script-datum-redeemer? or other?

Reply:
I meant the title at the top, the file name is okay. Maybe "How to export scripts, datums, and redeemers"

Reply:
Done